src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="input-form"
+      action="https://www.google.com"
+      method="post"
+      data-qa="big"
+    >
+      <span class="search-icon">
+        <input 
+          class="input"
+          placeholder="Try &quot;Los Angeles&quot;"
+          type="text"
+          data-qa="keypress"
+        >
+      </span>
+    </form>
+    <form 
+      class="input-form"
+      action="https://www.google.com"
+      method="post"
+      data-qa="small"
+    >
+      <span class="search-icon-small">
```

Comment:
why do u need extra tag?

Reply:
bc :before/:after is [not applicable](https://stackoverflow.com/questions/2587669/can-i-use-a-before-or-after-pseudo-element-on-an-input-field) to input tag

Reply:
okay, in that case, it is better to use `img` because u use an extra tag in both cases let's make it meaningful